tests/bootstrap/cli.php

```diff
@@ -26,6 +26,7 @@
 $frameworkDir = basename($frameworkPath ?? '');
 
 // Copied from cli-script.php, to enable same behaviour through phpunit runner.
+// This is all legacy and should be removed in a future major release.
```

Comment:
I would have just removed this outright but it's a bit late into the cycle to do that.